src/Grpc.AspNetCore.Server/InterceptorRegistration.cs

```diff
@@ -78,6 +78,10 @@ public class InterceptorRegistration
         private IGrpcInterceptorActivator? _interceptorActivator;
         private ObjectFactory? _factory;
 
+#if NET5_0_OR_GREATER
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
+            Justification = "Type parameter members are preserved with DynamicallyAccessedMembers on InterceptorRegistration.Type property.")]
```

Comment:
Hmm, I thought we fixed this in the trimmer, but I guess we only fixed it for [MakeGenericMethod](https://github.com/dotnet/linker/issues/1727) and not MakeGenericType.

@vitek-karas @MichalStrehovsky - do you know why we didn't do the same for MakeGenericType? In this case, `typeof(IGrpcInterceptorActivator<>)` doesn't have any trimming annotations on it. So calling `MakeGenericType` should be safe.

Or is this the case that the Trim Analyzer blindly injects this warning and doesn't have the same smarts as the real ILLinker?

Reply:
This is unfortunate. The analyzer has been fixed in https://github.com/dotnet/linker/pull/2209, but apparently this change didn't make it into the 6.0. It should be in 6.0.2xx though.

It is analyzer only problem - I would suggest we suppress this with #pragma instead of the attribute.